lib/elixir/lib/enum.ex

```diff
@@ -1181,7 +1181,7 @@ defmodule Enum do
   Returns the biggest of the elements in the enumerable according
   to Erlang's term ordering.
 
-  If more than one elements compare equal, the first one that was found
+  If multiple elements are considered the biggest, the first one that was found
```

Comment:
I think `biggest` reads strange and it isn't always true, maybe say `maximal`.
Same applies to `smallest` vs. `minimal`.